firebase/functions/src/comments.ts

```diff
@@ -68,7 +65,33 @@ export const new_comment = functions.https.onCall(async (data, context) => {
   };
   return await addToFIreBase(commentData, "Comments");
 });
- 
+
+export const update_location = functions.https.onCall(async (data, context) => {
+  const user_uid = context.auth ? context.auth.uid : ''
+  if (user_uid === '') {
+    return { res: '400', msg:'Usuario no existe'}
```

Comment:
There must be thrown an exception and console must be aware of unauthenticated user.
```suggestion
    const uid_ = context.auth.uid || null;
    console.error('[Update location] There was an error with uid:', uid_);
    throw new functions.https.HttpsError('unauthenticated', 'The function must be called while authenticated.')
```